packages/manager/src/features/Users/UsersLanding.tsx

```diff
@@ -19,88 +15,67 @@ import { useProfile } from 'src/queries/profile';
 
 import CreateUserDrawer from './CreateUserDrawer';
 import { UserDeleteConfirmationDialog } from './UserDeleteConfirmationDialog';
-import { UserRow } from './UserRow';
 import { UsersLandingProxyTableHead } from './UsersLandingProxyTableHead';
 import { UsersLandingTableBody } from './UsersLandingTableBody';
 import { UsersLandingTableHead } from './UsersLandingTableHead';
 
+import type { Filter } from '@linode/api-v4';
+
 export const UsersLanding = () => {
+  const [isCreateDrawerOpen, setIsCreateDrawerOpen] = React.useState<boolean>(
+    false
+  );
+  const [isDeleteDialogOpen, setIsDeleteDialogOpen] = React.useState(false);
+  const [selectedUsername, setSelectedUsername] = React.useState('');
   const flags = useFlags();
   const { data: profile } = useProfile();
 
   const pagination = usePagination(1, 'account-users');
   const order = useOrder();
 
+  const usersFilter: Filter = {
+    ['+order']: order.order,
+    ['+order_by']: order.orderBy,
+  };
+
+  if (flags.parentChildAccountAccess) {
+    usersFilter['user_type'] = { '+neq': 'proxy' };
+  }
+
   const { data: users, error, isLoading, refetch } = useAccountUsers(
     {
       page: pagination.page,
       page_size: pagination.pageSize,
     },
-    {
-      '+order': order.order,
-      '+order_by': order.orderBy,
-    }
+    usersFilter
+  );
+
+  const {
+    data: proxyUser,
+    error: proxyUserError,
+    isLoading: loadingProxyUser,
+  } = useAccountUsers(
+    {},
+    flags.parentChildAccountAccess ? { user_type: 'proxy' } : {}
```

Comment:
Maybe we should leave this as is

```suggestion
    { user_type: 'proxy' }
```

and disable the query if the flag is off?

Reply:
Yea that will be cleaner, I'll just have to update the query itself